spark/src/main/scala/io/delta/sharing/spark/RemoteDeltaLog.scala

```diff
@@ -293,7 +305,7 @@ private[sharing] case class RemoteDeltaFileIndex(
 
   override def refresh(): Unit = {}
 
-  override def sizeInBytes: Long = deltaLog.snapshot.sizeInBytes
```

Comment:
I have this change in my PR as well... so perhaps revert from here?
I also got rid of deltaLog arg from the file index.

Reply:
This still need it to be correct. I'll wait for your merge first.